frontend/src/modules/positions/components/NewPosition.jsx

```diff
@@ -1,42 +1,164 @@
 import React from "react"
 import { connect } from "react-redux"
 import { createNewPosition } from "../actions"
+import { fetchInstructors } from "../../instructors/actions"
 import { Grid, Col, Button } from "react-bootstrap"
+import moment from "moment"
```

Comment:
could we stick to convention and use `import { moment } from "moment"` ?

Reply:
Updated!